processor-samples/streamlistener-basic/README.adoc

```diff
@@ -66,6 +66,6 @@ All the instructions above apply here also, but instead of running the default `
 
 * `./mvnw clean package -P rabbit-binder`
 
-* `java -jar target/streamlistener-basic-0.0.1-SNAPSHOT.jar`
+* `java -jar target/streamlistener-basic-0.0.1-SNAPSHOT-rabbit.jar`
```

Comment:
I think this must be `streamlistener-basic-rabbit-0.0.1-SNAPSHOT.jar`.

No? What am I missing?

Reply:
mvn clean package -P rabbit-binder generate two jar files. 
streamlistener-basic-0.0.1-SNAPSHOT.jar and streamlistener-basic-0.0.1-SNAPSHOT-rabbit.jar

command: `java -jar target/streamlistener-basic-0.0.1-SNAPSHOT.jar`
couse error: 

> no main manifest attribute, in target/streamlistener-basic-0.0.1-SNAPSHOT.jar

For me only this command works `java -jar target/streamlistener-basic-0.0.1-SNAPSHOT-rabbit.jar`

Reply:
OK. My bad. So, let's leave it as you fixed.